PBN RCT.Rmd

```diff
@@ -27,157 +27,210 @@ if(!require(openxlsx)){install.packages('openxlsx')}
 original_survey_data <- original_survey_data %>%
   select(-Q_AmbiguousTextPresent, -Q_AmbiguousTextQuestions)
 
-##Now we can merge
+#Now we can merge
 merged_data <- bind_rows(original_survey_data, new_survey_data)
 
+#In the baseline data, select the participants being included in analysis (per data integrity checks, should be N=220)
+included_participants <- merged_data %>%
+  filter(Participant_ID %in% c(40077, 15186, 11697, 95674, 40889, 84608, 61437, 16432, 22485, 24378, 21140, 58776, 13649, 83679, 62516, 52326, 20457, 39455, 64886, 35756, 65322, 86883, 72424, 92104, 33504, 38713, 96986, 28301, 14617, 14806, 93863, 13842, 31849, 74139, 53775, 57712, 96120, 49399, 45841, 40397, 75164, 71675, 30277, 58650, 18169, 41917, 50884, 70371, 48434, 18974, 18856, 17231, 56049, 68213, 93764, 15333, 98726, 80569, 63962, 66481, 40376, 83329, 90314, 59978, 66920, 35067, 15025, 77121, 89229, 82287, 96711, 11308, 53611, 57517, 47321, 43097, 74462, 11290, 49908, 24314, 69311, 70500, 49864, 69898, 74951, 38601, 60126, 26551, 58830, 21592, 17149, 31067, 39743, 71511, 45789, 84417, 56534, 98233, 84267, 68200, 76371, 45565, 48031, 19978, 25987, 38488, 25643, 86804, 86300, 18348, 90947, 53863, 17032, 60501, 27455, 62781, 88241, 55765, 39606, 66912, 67236, 11868, 68267, 35553, 95389, 80941, 15777, 53969, 95004, 43258, 64019, 99949, 72626, 47331, 50401, 96007, 65187, 31188, 93980, 76870, 32309, 85411, 85344, 48970, 54988, 49034, 77719, 78040, 86763, 33280, 23158, 87771, 21842, 19199, 96320, 64719, 78881, 49236, 26869, 65509, 38538, 73614, 80888, 55845, 86114, 22584, 81514, 92289, 86314, 65863, 77450, 25638, 88046, 99374, 13638, 35047, 38572, 26776, 86023, 43295, 20973, 53220, 29614, 62193, 42215, 41799, 93664, 50466, 78161, 85350, 87127, 15065, 11827, 63743, 20364, 17670, 44268, 48593, 96225, 71632, 97410, 90795, 49102, 68198, 16855, 64531, 13302, 57955, 21313, 13023, 69913, 75263, 93022, 48022, 22684, 17052, 72038, 56590, 56815, 76432))
+
+#In the follow-up data, select the included participants who have completed follow-up 
+###WILL NEED TO UPDATE WITH FULL SAMPLE###
+followup_data <- followup_data %>%
+  filter(Participant_ID %in% c(40077, 15186, 11697, 95674,40889, 84608, 61437, 16432, 21140, 58776, 62516, 20457, 39455, 35756, 65322, 72424, 38713, 96986, 28301, 14617, 14806, 93863, 13842, 31849, 57712, 45841, 40397, 75164, 71675, 58650, 41917, 50884, 48434, 18856, 17231, 56049, 68213, 93764, 15333, 98726, 80569, 63962, 40376, 83329, 90314, 59978, 66920, 35067, 89229, 82287, 96711, 11308, 53611, 57517, 74462, 24314, 69898, 74951, 38601, 60126, 58830, 17149, 31067, 98233, 84267, 45565, 48031, 19978, 25987, 38488, 86804, 86300, 18348, 17032, 60501, 88241, 39606, 67236, 11868,68267, 35553, 95389))
+#As of 5/9/24 compensation, this should only be N=82, but there are 105 rows because of duplicates. Get rid of duplicates (keeping the one with the most Progress, and if Progress is 100 for multiple, keeping most recent by RecordedDate). 
+###NEED TO CHECK THIS WORK###
```

Comment:
It's a good idea to double check but this should work! You can also write it like this, which might be a little more intuitive (but should do the same thing)

```
followup_data_deduplicated <- followup_data %>%
  group_by(Participant_ID) %>%
  arrange(desc(Progress), RecordedDate) %>%
  slice(1)
```